Sdk/Speckle.Connectors.Utils/Connector.cs

```diff
@@ -18,42 +23,33 @@ public static class Connector
 
   public static HostApplication HostApp { get; private set; }
 
-  public static IDisposable? Initialize(HostApplication application, HostAppVersion version)
+  public static IDisposable? Initialize(
+    HostApplication application,
+    HostAppVersion version,
+    SpeckleContainerBuilder builder
+  )
   {
     Version = version;
     VersionString = HostApplications.GetVersion(version);
     HostApp = application;
     TypeLoader.Initialize(typeof(Base).Assembly, typeof(Point).Assembly);
 
-#if DEBUG || LOCAL
-    var config = new SpeckleConfiguration(
-      application,
-      version,
-      new(MinimumLevel: SpeckleLogLevel.Information, Console: true, File: new(Path: "SpeckleCoreLog.txt")),
-      new(Console: false, Otel: new())
-    );
-#else
-    var config = new SpeckleConfiguration(
-      application,
-      version,
-      new(
-        MinimumLevel: SpeckleLogLevel.Information,
-        Console: false,
-        File: new(Path: "SpeckleCoreLog.txt"),
-        Otel: new(
-          Endpoint: "https://seq-dev.speckle.systems/ingest/otlp/v1/logs",
-          Headers: new() { { "X-Seq-ApiKey", "y5YnBp12ZE1Czh4tzZWn" } }
-        )
-      ),
+    var (logging, tracing) = Observability.Initialize(
+      VersionString,
+      Slug,
+      Assembly.GetExecutingAssembly().GetVersion(),
       new(
-        Console: false,
-        Otel: new(
-          Endpoint: "https://seq-dev.speckle.systems/ingest/otlp/v1/traces",
-          Headers: new() { { "X-Seq-ApiKey", "y5YnBp12ZE1Czh4tzZWn" } }
-        )
+        new SpeckleLogging(Console: true, Otel: null, MinimumLevel: SpeckleLogLevel.Warning),
```

Comment:
Not logging Information logs?

Reply:
Otel: null?

Reply:
by default, currently, this is not logging anywhere yet